flink/src/main/java/org/apache/iceberg/flink/source/FlinkSource.java

```diff
@@ -70,6 +73,7 @@ public static Builder forRowData() {
     private Table table;
     private TableLoader tableLoader;
     private TableSchema projectedSchema;
+    private ReadableConfig readableConfig;
```

Comment:
Should we provide a `new Configuration()`  for this variable ?  Otherwise,  it will just throw NPE if people forget to provide a `flinkConf` in FlinkSource#Builder because we don't check the nullable in `interParallelism`.

Reply:
yes,I add the `new Configuration()` for default.